smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/endpoints/EndpointResolverGenerator.kt

```diff
@@ -62,4 +63,17 @@ class EndpointResolverGenerator(
         writer.write("")
         writer.write("extension DefaultEndpointResolver: EndpointResolver {}")
     }
+
+    private fun renderStaticResolver(writer: SwiftWriter) {
```

Comment:
Static Resolver will be used by tests but are also part of SRA to allow users to bypass endpoint resolution and provide their own endpoint